documentation/SOFTWARE_DESIGN.md

```diff
@@ -65,7 +65,9 @@ Sokoban constructor(const vec<vec<string>> levels)
 // class properties
 const vec<vec<string>> levels, read-only
 uint current_level // index into levels
-uint moves
+vector<Direction> moves // history of moves made by the player
+unit player_x // the player's column position
```

Comment:
```suggestion
unit player_x // the player's column position
```
uint player_x // the player's column position

Reply:
`uint` means "unsigned integer", sorry to be unclear!